src/OrbitGl/BasicPageFaultsTrack.cpp

```diff
@@ -35,9 +35,7 @@ BasicPageFaultsTrack::BasicPageFaultsTrack(Track* parent,
       AnnotationTrack(),
       cgroup_name_(std::move(cgroup_name)),
       memory_sampling_period_ms_(memory_sampling_period_ms),
-      parent_(parent) {
-  draw_background_ = false;
```

Comment:
Was this removed on purpose?

Reply:
Yes, it was, like Pablo explained - it was only used when a track was a child of another track. Which is also the only reason why indentation exists, so I folded both properties into one.

Reply:
Thanks. I missed the fact that we are using intendationlevel now.